src/components/cars/ManageBookingCard.tsx

```diff
@@ -0,0 +1,103 @@
+import axios from 'axios'
+import { CalendarIcon, TimeIcon } from '../../assets'
+import { BookingState, BookingWithReferences } from '../../util/api'
+import { formatBookingStatus } from '../../util/booking'
+import { convertMonth, timeFormatter } from '../../util/date'
+import ButtonComponent from '../ui/Button'
```

Comment:
I think You need to pull from dev and merge to remove the buttonComponent
```suggestion
import Button from '../ui/Button'
```